README.md

```diff
@@ -1,6 +1,13 @@
-# Give Lively Lint
+# Give Lively Lint runner
+
+`GLLint` is a Ruby gem that handles running linters against specific files (e.g. just the changed files from a branch). It leverages `git`, `rubocop` and `prettier`
```

Comment:
Nit: but doesn't it also run `eslint`?

Reply:
You're correct, thank you 🙇! This should be eslint - this gem uses eslint, not prettier.

I replaced the reference to prettier, since they were incorrect.